test/ts-parser.test.js

```diff
@@ -5,24 +5,43 @@ var expect = require('chai').expect;
 var TSParser = require('../lib/tsParser');
 
 describe('TypeScript Parser Test', function() {
-  var tsOptions = {tsconfig: path.join(__dirname, 'tsconfig.json')};
+  var tsconfig = path.join(__dirname, 'tsconfig.json');
 
-  it('should parse this TS file', function() {
-    this.timeout(90000); // typedoc can be time consuming
+  it('should parse TS file', function() {
     var file = path.join(__dirname, 'fixtures/ts/Greeter.ts');
     var tsFiles = [file];
-    var tsParser = new TSParser(tsFiles, tsOptions);
+    var tsParser = new TSParser(tsFiles, {
+      excludeNotExported: false,
+      tsconfig,
+    });
     var parsedData = tsParser.parse();
     expect(parsedData.sections).to.have.length(3);
     expect(parsedData.constructs).to.have.length(1);
     expect(parsedData.errors).to.have.length(0);
   });
 
+  it.skip('should parse TS file for exported ' +
```

Comment:
"should parse TS file for exported" - looks like an incomplete sentence to me, for exported _what_?

It took me a while to figure out why this test is marked as `skipped`, I assume it requires a feature that's not supported by typedoc yet. Is that correct? In which case, please add a regular code comment explaining the situation and pointing to the typedoc issue.

```js
// Typedoc does not support this feature yet
// See https://github.com/TypeStrong/typedoc/pull/694
it.skip('should parse TS file for exported <what> - not supported by typedoc yet', function() {
  // ...
});
```